ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml

```diff
@@ -108,11 +108,10 @@ SettingsPageLayout {
                 model: root.membersModel
                 rootStore: root.rootStore
                 placeholderText: {
-                    if (root.membersModel.count === 0) {
+                    if (root.membersModel.count === 0)
```

Comment:
Can leave as is here but just wanna point out that it's rather bad practice to remove these parens in JS, even if the resulting statement is a single line

Reply:
Yeah I see... it's a common practice in C++ to have single line statements w/o the braces